### DISCUSSION TITLE
Reopening  #3778 Version Bump Latest version dependencies do not resolve.

### DISCUSSION THREAD

**Discussion by tracure1337** [ansible/ansible-lint#3805] [Q&A]
Was closed with comment poetry is not supported but has nothing to do with the issue at hand.

The issue would happen in every package manager since the versions do not resolve.

Using version ^6.20.0 for ansible-lint
```

Updating dependencies
Resolving dependencies... (0.3s)

Because no versions of ansible-lint match >6.20.0,<7.0.0
 and ansible-lint (6.20.0) depends on will-not-work-on-windows-try-from-wsl-instead (*), ansible-lint (>=6.20.0,<7.0.0) requires will-not-work-on-windows-try-from-wsl-instead (*).
So, because no versions of will-not-work-on-windows-try-from-wsl-instead match *
 and severaln1nes depends on ansible-lint (^6.20.0), version solving failed.
```

Thanks for looking into this.

**Comment by ssbarnea** [comment]
Please do not abuse the issue tracker for support. I already mentioned that poetry is not supported and that is also documented in official documentation.

You also did not complete the information required in a ticket, like full commands executed. Only by chance I was able to guess that you used poetry due to the use of `^` for version constraints.
